docs/pages/installation/kinD.md

```diff
@@ -153,9 +156,9 @@ Run the following:
 
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- $ git clone https://github.com/layer5io/meshery.git; cd meshery
- $ kubectl create namespace meshery
- $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
+$ kubectl create ns meshery
+$ helm repo add meshery https://meshery.io/charts/
+$ helm install meshery meshery/meshery -n meshery

```

Comment:
These steps are just fine... no need to give `--create-namespace` option too as Helm will automatically create a namespace with above two lines alone

Reply:
But however...
```suggestion
$ helm repo add meshery https://meshery.io/charts/
$ helm install meshery meshery/meshery --namespace meshery

```
Give this line, as `-n` expects a namespace already made. By replacing with `--namespace` it generates it